sdk/translation/Azure.AI.Translation.Document/README.md

```diff
@@ -81,6 +81,27 @@ string apiKey = "<apiKey>";
 var client = new DocumentTranslationClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
 ```
 
+#### Create DocumentTranslationClient with Azure Active Directory Credential
+
+Client API key authentication is used in most of the examples in this getting started guide, but you can also authenticate with Azure Active Directory using the [Azure Identity library][azure_identity].  Note that regional endpoints do not support AAD authentication.
+
+Create a [custom subdomain][custom_subdomain] for your resource in order to use this type of authentication.  
+
+To use the [DefaultAzureCredential][DefaultAzureCredential] provider shown below, or other credential providers provided with the Azure SDK, please install the Azure.Identity package:
```

Comment:
check the links here, doesn't seem like they exist

Reply:
double plop.
Thank you! added